src/protocols/secure_channel/CASEServer.h

```diff
@@ -72,7 +85,16 @@ class CASEServer : public SessionEstablishmentDelegate,
 
     CHIP_ERROR InitCASEHandshake(Messaging::ExchangeContext * ec);
 
-    void Cleanup();
+    /*
+     * This will clean up any state from a previous session establishment
+     * attempt (if any) and setup the machinery to listen for and handle
+     * any session handshakes there-after.
+     *
+     * If a session had previously been established successfully, previouslyEstablishedPeer
+     * should be set to the scoped node-id of the peer associated with that session.
+     *
+     */
+    void Cleanup(ScopedNodeId previouslyEstablishedPeer = ScopedNodeId());
```

Comment:
Still think this should be 
```suggestion
    void Cleanup(const ScopedNodeId & previouslyEstablishedPeer = ScopedNodeId());
```
so we don't pass 2-word things around by value....